src/app/components/put-away/put-away.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export default function PutAway(
+    {onClick, ariaLabel='dismiss'}: {onClick: () => void, ariaLabel?: string}
+) {
+    const closeUsingKeyboard = React.useCallback(
+        (event: React.KeyboardEvent) => {
+            if ([' ', 'Escape', 'Enter'].includes(event.key)) {
+                onClick();
+                event.preventDefault();
+                event.stopPropagation();
+            }
+        },
+        [onClick]
+    );
+
+    return (
+        <span
+            className='put-away'
+            aria-label={ariaLabel}
+            role='button'
+            onClick={onClick}
+            tabIndex={0}
+            onKeyDown={closeUsingKeyboard}
+            data-nudge-action='dismissed'
+        >
+            &times;
```

Comment:
Don't you want to keep the FontAwesome icon for this?

Reply:
I had some with FontAwesome and some with `&times`. They should be consistent, so I picked `&times`.

Reply:
I'd personally pick the FontAwesome to standardize on, because it's less prone to font rendering differences/issues. I've found the svgs to be much easier to keep centered in buttons and other ui elements.